persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/SampleListMyBatisRepository.java

```diff
@@ -66,6 +68,7 @@ public BaseMeta getMetaSampleListsInStudy(String studyId) {
     }
 
     @Override
+    @Cacheable(cacheResolver = "staticRepositoryCacheOneResolver", condition = "@cacheEnabledConfig.getEnabled()")
```

Comment:
@jagnathan i'm confused. won't produce cache keyed by the LIST of sample ids instead of inidvidual sampleids?

Reply:
The input is list of sampleListId. But the list will always have a single entry. This function is called from service/src/main/java/org/cbioportal/service/impl/SampleListServiceImpl.java - addSampleIds() method.